trackpy/linking/linking.py

```diff
@@ -503,6 +500,9 @@ def next_level(self, coords, t, extra_data=None):
     def assign_links(self):
         spl, dpl = [], []
         for source_set, dest_set in self.subnets:
+            for sp in source_set:
+                sp.forward_cands.sort(key=lambda x: x[1])
+
```

Comment:
Am I correct with assuming that not every subnetlinker requires the null particle? Else it's probably best done here?